graphql/ui_plugin_build_baron.go

```diff
@@ -121,27 +121,24 @@ type buildBaronConfig struct {
 }
 
 func GetSearchReturnInfo(taskId string, exec string) (*thirdparty.SearchReturnInfo, buildBaronConfig, error) {
-	bbConfig := &buildBaronConfig{}
+	bbConfig := buildBaronConfig{}
 	t, err := BbGetTask(taskId, exec)
 	if err != nil {
-		return nil, *bbConfig, err
+		return nil, bbConfig, err
 	}
 	settings := evergreen.GetEnvironment().Settings()
 	buildBaronProjects := BbGetConfig(settings)
 	bbProj, ok := buildBaronProjects[t.Project]
 
 	if !ok {
-		bbConfig.ProjectFound = false
-		return nil, *bbConfig, errors.New(fmt.Sprintf("Build Baron project for %s not found", t.Project))
+		return nil, bbConfig, errors.Errorf("Build Baron project for %s not found", t.Project)
 	}
 	bbConfig.ProjectFound = true
 
 	// the build baron is configured if the jira search is configured
-	bbSearchConfigured := len(bbProj.TicketSearchProjects) > 0
-
-	if !bbSearchConfigured {
+	if !(len(bbProj.TicketSearchProjects) > 0) {
```

Comment:
nit: rather than negating the whole expression, it'll be more readable to flip > to <= 0